Code/Editor/Plugins/ProjectSettingsTool/PlatformSettings_Android.cpp

```diff
@@ -209,12 +209,10 @@ namespace ProjectSettingsTool
                         ->Attribute(Attributes::FuncValidator, ConvertFunctorToVoid(&Validators::PackageName))
                         ->Attribute(Attributes::LinkOptional, true)
                         ->Attribute(Attributes::PropertyIdentfier, Identfiers::AndroidPackageName)
-                        ->Attribute(Attributes::LinkedProperty, Identfiers::IosBundleIdentifer)
```

Comment:
FYI: if we are removing the link between these properties, the whole linked property system inside the Project Settings Tool can be removed as well.

Reply:
\yes, of course. Done.